src/screens/Game/GameSubMenu/index.tsx

```diff
@@ -248,51 +248,53 @@ export default function GamesSubmenu({
   }
 
   useEffect(() => {
-    if (isWin) {
-      return
-    }
-    const getWineInfo = async () => {
-      try {
-        const { wineVersion, winePrefix }: AppSettings =
-          await ipcRenderer.invoke('requestSettings', appName)
-        let wine = wineVersion.name
-          .replace('Wine - ', '')
-          .replace('Proton - ', '')
-        if (wine.includes('Default')) {
-          wine = wine.split('-')[0]
+    if (isInstalled) {
+      if (isWin) {
```

Comment:
Could've done something like
```suggestion
    if (!isInstalled || isWin) {
```
here to avoid the extra indentation if that's desired

Reply:
Yes good catch.